src/rdb/README.md

```diff
@@ -19,11 +19,11 @@ The <a href="#f8.1">figure</a> below shows the modules constituting a service re
 <a id="8.3.2"></a>
 ## RPC Handling
 
-When an RPC request arrives at the leader, a service thread of the service module picks up the request and handles it by executing a handler function designed for this type of request. As far as service replication is concerned, a handler comprises state queries (e.g., reading the epoch state), state updates (writing a new version of the pool map), and RPCs to other services (e.g., TARGET_CONTAINER_OPEN RPCs sent to target services). Some handlers involve only queries, some involve updates as well as queries, and others involve all three kinds of actions; rarely, if ever, do handlers involve only updates but no queries.
+When an RPC request arrives at the leader, a service thread of the service module picks up the request and handles it by executing a handler function designed for this type of request. As far as service replication is concerned, a handler comprises state queries (e.g., reading the pool properties), state updates (writing a new version of the pool map), and RPCs to other services (e.g., CONT_TGT_DESTROY RPCs sent to target services). Some handlers involve only queries, some involve updates as well as queries, and others involve all three kinds of actions; rarely, if ever, do handlers involve only updates but no queries.
```

Comment:
very minor suggestion, but only for consistency within the other examples within the same sentence:
change from:
(writing a new version of the pool map)
to:
(e.g., writing a new version of the pool map)

Reply:
Fixed. Thanks.